kimchi/src/tests/foreign_field_mul.rs

```diff
@@ -1196,15 +1110,15 @@ fn test_random_multiplicands_carry1_lo() {
             &left_input,
             &right_input,
             &secp256k1_modulus(),
-            vec![((0, 4), PallasField::one())], // Invalidate carry1_lo
+            vec![((0, 6), PallasField::one())], // Invalidate carry1_lo
```

Comment:
Mmm is it really a 6 here? It used to be 4 before, seems like in other tests you only need to decrease by 1.

Reply:
Yeah, it's in 6
```
//~ |   6 | `carry1_lo`           (copy) |
```
That was a bug that is now fixed.

Reply:
Aaaah so the bug was there before! good good